docs/architecture/adr/0028-adopt-ngrx-signals-for-component-state-stores.md

```diff
@@ -0,0 +1,110 @@
+---
+adr: "0028"
+status: "Proposed"
+date: 2025-07-08
+tags: [client]
+---
+
+# 0028 - Adopt `@ngrx/signals` for Component State Stores
```

Comment:
🎨 Can your code reference be simplified somehow? Maybe just say "Angular Signals"? A lot of title usages can't render the preformatted text, but you can use the code in paragraphs.